docs/services/ml/deploy/tflite_cpu.md

```diff
@@ -158,25 +158,17 @@ You can use one of these architectures or build your own.
 
 ## Next steps
 
-To use your model with your machine, add a [vision service](/services/vision/) or a {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}}:
+The ML model service only runs your model on the machine.
+To use the inferences from the model, you must use an additional service such as a [vision service](/services/vision/):
 
 {{< cards >}}
 
-{{% manualcard link="/services/vision/mlmodel/"%}}
+{{% manualcard link="/services/vision/mlmodel/" title="Create a visual detector or classifier" noimage="True" %}}
 
-<h4>Create a detector with your model</h4>
-
-Configure an `mlmodel detector`.
-
-{{% /manualcard %}}
-{{% manualcard link="/services/vision/mlmodel/"%}}
-
-<h4>Create a classifier with your model</h4>
-
-Configure your `mlmodel classifier`.
+Use your model deployed with the ML model service by adding a vision service that can provide detections or classifications depending on your ML model.
```

Comment:
```suggestion
Use your model deployed with the ML model service by adding a vision service that can provide detections, classifications, or segmentations depending on your ML model.
```
not sure if accurate-- would this also include point clouds?

Reply:
I think I investigated this before and it's only detections and classifications but not 100% sure